org.eclipse.jdt.ls.tests/src/org/eclipse/jdt/ls/core/internal/handlers/WorkspaceSymbolHandlerTest.java

```diff
@@ -187,6 +187,17 @@ public void testSearchQualifiedTypeWithWildcards() {
 		assertTrue(results.stream().allMatch(s -> s.getName().endsWith("Exception") && s.getContainerName().contains(".lang")));
 	}
 
+	@Test
+	public void testSearchAllTypesOfPackage() {
+		List<SymbolInformation> results = WorkspaceSymbolHandler.search("java.io", monitor);
+		assertTrue(results.size() > 1);
+		assertTrue(results.stream().anyMatch(s -> "File".equals(s.getName()) && "java.io".equals(s.getContainerName())));
+
+		results = WorkspaceSymbolHandler.search("java.lang", monitor);
+		assertTrue(results.size() > 1);
+		assertTrue(results.stream().anyMatch(s -> s.getName().endsWith("Exception") && "java.lang".equals(s.getContainerName())));
```

Comment:
Oops, this is a bit wrong when I copied it over. Can just be equals rather than endsWith.

Reply:
Done.